lib/firebase.js

```diff
@@ -1,13 +1,15 @@
-import { Firebase } from "firebase/app";
+import { Firebase, initializeApp } from "firebase/app";
 import "firebase/auth";
+import { getAuth } from "firebase/auth";
 const FirebaseCredentials = {
-  apiKey: process.env.NEXT_PUBLIC_FIREBASE_PUBLIC_API_KEY,
-  authDomain: process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN,
-  projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
+  apiKey: "AIzaSyC_PhnQjxwx3ufu9OpiKLzP_jkEceMQ3JU",
```

Comment:
To keep it secure, you can create .env.local file and add these values in there 
<img width="832" alt="image" src="https://github.com/rfueston/kopla/assets/125618530/52c6710f-8270-47ea-8099-34645b6e2e90">

Reply:
gitignore file will ignore that file.

Reply:
thank you! Yes I keep forgetting to not push our API key info. That's my fault